contents/blog/sales-no-experience.md

```diff
@@ -0,0 +1,142 @@
+
+## One job, people
+
+You have one job. Get to product market fit.
+
+It's a _terrible_ idea to out
+
+It _could_ of course work, but it just adds another variable to the most important and most multivariate problem you will _ever_ solve as a startup.
+
+## A little pep talk
+
+_You_ are the best salesperson for your product. There will _never_ be anyone else as well qualified to sell.
+
+* You get the _why_ behind what you're building
+* You are the parent of your startup (TODO INSERT LINK TO AIRBNB FOUNDER MODE BIT ON THIS). This means you are able to tweak things - like shipping an extra feature on the spot, your pricing, or how you pitch the product.
+
+You need to prioritize figuring out making money as one of the very top things:
+
+* do users like your product
+* do users pay for it
+
+Everything else is a distraction early on. So what are you doing if you don't care about this? Likely being avoidant from all the rejection and fear of failure! You may not want to hear it but a magical salesperson will be _far_ less likely to figure this out than you.
+
+## Getting meetings
+
+Things that worked to recruit users:
+
+* Friends
+* People you've worked with before
+* Posting on your social media
+* LinkedIn DMs - customized
+* Cold emails - customized
+* Posting our startup to places where our users hang out (like HackerNews for PostHog), in a non spammy way
+
+Things that didn't really work:
+
+* Going to events - this just felt awkward and spammy
+* Automated emails or DMs - you aren't trying to scale up, you are trying to get just a few paying customers early on, so don't try to scale up your outreach, focus on the quality early on
+
+## Running a sales meeting
+
+As a more technical person, the first thing to be clear on is that this is a _sales_ meeting not a _product_ meeting.
+
+In _both_ types of meeting, you should be asking lots of questions. However, there are some key differences:
+
+* In sales meetings, you are aiming to _close down_ objections. In product meetings you want to _expand_ on them.
+* In sales meetings, you are steering the customer. In product meetings, you should let users meander.
+* In sales meetings, you learn by seeing if it closes or not. In product meetings, you learn based on what people are saying.
+
+We should talk vibes for a moment. First, don't make it painful for the customer to interact with you. _You_ are leading the meeting because you asked forit. A good way of doing this is to have some friendly rapport at the beginning, then layout "hey so there are four things I hoped to run through with you in this meeting. First, I want to get a grip on your company and priorities, second I'd like to cover X problem space a bit, and third, I wanted to talk about our product and show you a few relevant bits of it. Is that a good use of the next 25 minutes from your perspective?" Asking for _permission_ to ask questions means it'll feel less awkward when you do. Because you should.
```

Comment:
```suggestion
We should talk vibes for a moment. First, don't make it painful for the customer to interact with you. _You_ are leading the meeting because you asked for it. A good way of doing this is to have some friendly rapport at the beginning, then layout "hey so there are four things I hoped to run through with you in this meeting. First, I want to get a grip on your company and priorities, second I'd like to cover X problem space a bit, and third, I wanted to talk about our product and show you a few relevant bits of it. Is that a good use of the next 25 minutes from your perspective?" Asking for _permission_ to ask questions means it'll feel less awkward when you do. Because you should.
```